packages/features/insights/server/virtual-queues.ts

```diff
@@ -73,6 +73,7 @@ class VirtualQueuesInsights {
             AND m.role = 'ADMIN'
           )
         )
+        AND (t."rrTimestampBasis" IS NULL OR t."rrTimestampBasis" = 'CREATED_AT')
```

Comment:
Router position doesn't work if the distribution basis is not CREATED_AT, as the result depends on the slot that is picked

Reply:
Would there ever be a case where the `rrTimestampBasis` is `START_TIME` when trying to get the form responses here? What impact will this have if we return nothing from this array?

Reply:
Not how router position works now. If an empty array is returned the dropdown in router position stays empty which is fine